thrall/app/lib/ElasticSearch.scala

```diff
@@ -36,31 +36,33 @@ object ElasticSearch extends ElasticSearchClient with ImageFields {
 
   def currentIsoDateString = printDateTime(new DateTime())
 
-  def indexImage(id: String, image: JsValue)(implicit ex: ExecutionContext): Future[UpdateResponse] = {
-    prepareImageUpdate(id)
+  def indexImage(id: String, image: JsValue)(implicit ex: ExecutionContext): List[Future[UpdateResponse]] = {
+    prepareImageUpdate(id).map( request => request
```

Comment:
We could make `prepareImageUpdate` to take a block to run on all indices.
e.g.
```Scala
prepareImageUpdate(id) { request =>
   // ...
}

// ...

def prepareImageUpdate(op: UpdateRequestBuilder => ActionResponse): UpdateRequestBuilder =
  getCurrentIndices.map(_.map (index =>
      val updateRequest = client.prepareUpdate(index, imageType, id)
        .setScriptLang("groovy")
      )
      op(request)
    ).get
```

It's a little hard to do this in GH.